docs/release/cut-sw-release.md

```diff
@@ -199,27 +225,15 @@ Once the web page is updated, run the following command to update the CA certifi
 verify that the version of the CA certificates match the version that was promoted to release.
 
 ```bash
+# moria.cs.wisc.edu
 /p/vdt/workspace/tarball-client/current/amd64_rhel7/osgrun osg-update-data
```

Comment:
Can you make a ticket to get rid of this step or change it so that it doesn't require moria?